python/aigverse/abc/_binary.py

```
@@ -0,0 +1,256 @@
+        The trimmed output of ABC's ``version`` command.
+
+    Raises:
+        AbcNotFoundError: If no ABC executable could be located.
+        AbcTimeoutError: If ABC did not terminate within ``timeout`` seconds.
+    """
+    binary = abc_binary()
+    # ABC drops an `abc.history` file into its working directory on every run,
+    # so keep it out of whatever directory the caller happens to be in.
+    try:
+        with tempfile.TemporaryDirectory(prefix="aigverse-abc-") as scratch:
+            completed = subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)